targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/TARGET_FRDM/PeripheralPins.c

```diff
@@ -44,6 +44,8 @@ const PinMap PinMap_ADC[] = {
     {PTC9,  ADC1_SE5b, 0},
     {PTC10, ADC1_SE6b, 0},
     {PTC11, ADC1_SE7b, 0},
+	{PTB4 , ADC1_SE10, 0},
```

Comment:
Small fix needed - code alignment - pins should be aligned

Reply:
Sorry I have a different setting of my code editor. Fix in bd59a4c.